python/src/numerous/_client.py

```diff
@@ -0,0 +1,68 @@
+"""GraphQL client wrapper for numerous."""
+
+import asyncio
+import os
+from typing import Optional, Union
+
+from numerous.generated.graphql.client import Client as GQLCleint
+from numerous.generated.graphql.fragments import CollectionNotFound, CollectionReference
+
+
+API_URL_NOT_SET="NUMEROUS_API_URL environment variable is not set"
+MESSAGE_NOT_SET = "NUMEROUS_API_ACCESS_TOKEN environment variable is not set"
```

Comment:
This is not well formatted. Try to run ruff format on the codebase. We should add a linter to check this as well, but I'll do that separately later :)